packages/js-sdk/package.json

```diff
@@ -0,0 +1,64 @@
+{
+	"name": "@directus/sdk",
+	"version": "10.3.3",
+	"description": "The official Directus SDK for use in JavaScript!",
+	"bugs": {
+		"url": "https://github.com/directus/directus/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/directus/directus.git",
+		"directory": "packages/js-sdk"
+	},
+	"main": "dist/sdk.cjs.js",
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"import": {
+				"node": "./index.mjs",
+				"default": "./dist/sdk.bundler.js"
+			},
+			"require": "./dist/sdk.cjs.js"
+		},
+		"./package.json": "./package.json"
+	},
+	"module": "dist/sdk.bundler.js",
+	"unpkg": "dist/sdk.esm.min.js",
+	"types": "dist/index.d.ts",
+	"files": [
+		"dist",
+		"index.mjs"
+	],
+	"scripts": {
+		"build": "tsc --project tsconfig.prod.json",
```

Comment:
Wondering if we should use `unbuild` for this! Maybe another issue 😁

Reply:
Yeah, might be a good fit! I think it makes sense to switch in a separate PR 👍